FreeRTOS/Test/CMock/queue/queue_utest_common.c

```diff
@@ -0,0 +1,324 @@
+/*
```

Comment:
Which functions would be helpful to share between tests?

Reply:
I was referring more to the common stubs like configASSERT, malloc/failed checks, which could be consistent across all test groups. However this can be done at a later point in time once all tests are stabilized.